### PR TITLE
corrected BagIT API Call

### LINKED ISSUE #9090: [BagIt Export] Correction on BagIT API Calls
I wanted to draw atttention on this section of Dataverse_Manual (link below):
https://guides.dataverse.org/en/latest/installation/config.html#id160
Instead the command line should look like this: (validated for localfileArchive)
curl -X POST -H "X-Dataverse-key: API_KEY" 
http://localhost:8080/api/admin/submitDatasetVersionToArchive/:persistentID/Version_Num?persistentID=doi-of_dataset

Also a reference from the dataverse-google-group:
https://groups.google.com/g/dataverse-community/c/UdGIaPjmKGA/m/2qOwImRIBgAJ

### PR DESCRIPTION
**Pull request by shlake**
**What this PR does / why we need it**:
corrects BagIT API Call which had the {verslon} parameter in the wrong place in the example

**Which issue(s) this PR closes**: #9090 9090

Closes #9090

**Additional documentation**:
Also fixed the bulleted list in this section describing the archive command, batch version's, parameters

### CODE REVIEW

File: doc/sphinx-guides/source/installation/config.rst [hunk 1]
```diff
@@ -1240,14 +1240,19 @@ API Calls
 
 Once this configuration is complete, you, as a user with the *PublishDataset* permission, should be able to use the admin API call to manually submit a DatasetVersion for processing:
 
-``curl -X POST -H "X-Dataverse-key: <key>" http://localhost:8080/api/admin/submitDatasetVersionToArchive/{id}/{version}``
+``curl -X POST -H "X-Dataverse-key: <key>" http://localhost:8080/api/admin/submitDatasetVersionToArchive/{version}/{id}``
```
Comment (qqmyers) [inline doc/sphinx-guides/source/installation/config.rst:1243]: This isn't right - the original is correct. Per https://guides.dataverse.org/en/latest/api/native-api.html?highlight=persistentid#get-json-representation-of-a-dataset , the ":persistentId" string is going in the {id} slot in the URL.

**Review by qqmyers — CHANGES_REQUESTED** [review]
Other than needing to revert line 1243, this looks good to me.

File: doc/sphinx-guides/source/installation/config.rst (hunk 1 above)
Comment (pdurbin) [reply doc/sphinx-guides/source/installation/config.rst:1243]: Right, on the Java side we have this:

    @Path("/submitDatasetVersionToArchive/{id}/{version}")

I'll revert the doc change. Good catch, @qqmyers.

**Review by pdurbin — APPROVED** [review]
@shlake thanks for the pull request! I reverted the thing @qqmyers noticed and made a few more tweaks. We're good now. Merging.

**Pull request merged**